--- /dev/fd/63	2021-05-17 21:35:56.529973277 +0100
+++ masterdir/builddir/389-ds-base-389-ds-base-2.0.4/dirsrvtests/tests/suites/resource_limits/fdlimits_test.py	2021-04-07 08:46:28.000000000 +0100
@@ -15,7 +15,7 @@
 
 FD_ATTR = "nsslapd-maxdescriptors"
 GLOBAL_LIMIT = resource.getrlimit(resource.RLIMIT_NOFILE)[1]
-SYSTEMD_LIMIT = ensure_str(check_output("sv show -p LimitNOFILE dirsrv@standalone1".split(" ")).strip()).split('=')[1]
+SYSTEMD_LIMIT = ensure_str(check_output("systemctl show -p LimitNOFILE dirsrv@standalone1".split(" ")).strip()).split('=')[1]
 CUSTOM_VAL = str(int(SYSTEMD_LIMIT) - 10)
 TOO_HIGH_VAL = str(GLOBAL_LIMIT * 2)
 TOO_HIGH_VAL2 = str(int(SYSTEMD_LIMIT) * 2)
