--- /dev/fd/63	2021-05-17 21:27:02.900238603 +0100
+++ masterdir/builddir/389-ds-base-389-ds-base-2.0.4/dirsrvtests/tests/suites/tls/tls_cert_namespace_test.py	2021-04-07 08:46:28.000000000 +0100
@@ -56,7 +56,7 @@
     standalone.enable_tls()
 
     log.info('Checking PrivateTmp value')
-    cmdline = ['sv', 'show', '-p', 'PrivateTmp', 'dirsrv@{}.service'.format(standalone.serverid)]
+    cmdline = ['systemctl', 'show', '-p', 'PrivateTmp', 'dirsrv@{}.service'.format(standalone.serverid)]
     log.info('Command used : %s' % format_cmd_list(cmdline))
     result = subprocess.check_output(cmdline)
     assert PRIVATE_TMP in ensure_str(result)
